<launch>
    <rosparam command="load" file="$(find eq_lio)/config/urbannav.yaml" />
    <node pkg="eq_lio" type="ieskf_frontend_node" name="ieskf_frontend_node" output="screen" /> 
	<node  pkg="rviz" type="rviz" name="rviz" args="-d $(find eq_lio)/rviz/show.rviz" /> 
</launch>
